### DISCUSSION TITLE
Y-Stepper makes weird noises

### DISCUSSION THREAD

**Discussion by JNSMDT** [bassamanator/Sovol-SV06-firmware#145] [Q&A]
Hey Everybody and Bassam,

first excellent work, this guide is really nice. 

I successfully installed klipper and everything works fine but, since i klipperized my SV06 Plus my y-Stepper Motor makes some weird noise. It's a little bit like a small fan is blowing inside the stepper.

Can anybody tell me if this is normal or should i lower the current/do anything else.

**Comment by bassamanator** [comment]
My friend, stepper motor settings are stock except for `microsteps`. You can try reducing them down to 64, 32, or 16. Keep in mind that printers will make different sounds based on these settings, however, that doesn't mean that one setting is wrong and the other one is correct.

It's hard to capture sounds well unless you have a professional setup, but it would help to have a listen to this noise.

**Reply by JNSMDT** [reply]
Hey,

I will try adjusting the steps tomorrow since I can't access the printer today.

Also found a video (not mine) from a guy with the same problem, if it is a problem at all. But in the short video you can here the noise which is pretty similar to mine

[YouTube Video](https://youtube.com/shorts/YI3HeZsbMRM?si=GbJ08qtiLFXWZ3EH)

**Reply by JNSMDT** [reply]
Hey me again,

so i lowered the microsteps to 16, which made the noise quieter but it's still there.

**Comment by bassamanator** [comment]
I know that sound, that's the hold_current sound. I imagine the sound stops when you do a [M84](https://marlinfw.org/docs/gcode/M018.html)?

Try this, under `[tmc2209 stepper_y]` add `interpolate: True`.

**Reply by JNSMDT** [reply]
yep when i send M84 the sound goes away, but `interpolate: True` doesn't help at all. Since it is only after homing and goes away/is not noticeable anymore while printing it is ok for now. My question was anyway more about potential damage then the the noise.

Thanks for the help

**Reply by bassamanator** [reply]
The only thing that you can change in the `printer.cfg` that could damage the stepper is too much current. The sound is the 'on' sound, while the motor is engaged, and it can be louder is some steppers, unfortunately.

Welcome.
